avocado/core/test.py

```diff
@@ -702,7 +703,9 @@ class SimpleTest(Test):
     def __init__(self, name, params=None, base_logdir=None, job=None):
         super(SimpleTest, self).__init__(name=name, params=params,
                                          base_logdir=base_logdir, job=job)
-        self._command = self.filename
+        self._command = None
+        if self.filename is not None:
+            self._command = pipes.quote(self.filename)
```

Comment:
Yes, let's keep at least the external runner with args. As you said, advanced features should be powerful. We can always re-evaluate that if it becomes a problem.